README.md

```diff
@@ -267,6 +267,16 @@ docker-compose exec web python3 manage.py makemigrations --help
 docker-compose exec web python3 manage.py migrate --help
 ```
 
+### Practical example
+Note: prepend `docker-compose exec web ` to the following python calls if your app is containerized.
+
```

Comment:
As this is project specific "rules", and because they were useful for me, I would keep these lines here.    
I'm also thinking of newcomers to the project, e.g. external academic partners...